src/app/styles/index.scss

```diff
@@ -15,7 +15,79 @@
 }
 
 @layer components {
-    .h6 {
-        @apply font-rubik font-light text-2xl
-     }
+.primary {
+    @apply font-rubik font-normal text-xl
+ }
+
+ .primary-bold {
+    @apply font-rubik font-semibold text-[22px]
+ }
+
+ .additional {
+    @apply font-rubik font-normal text-base
+ }
+
+ .h6 {
+    @apply font-rubik font-light text-2xl
+ }
+}
+
+
+$backgroundImageColored: "data:image/svg+xml;utf8,<svg width='24' height='24' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path d='M12 2L14.2451 8.90983H21.5106L15.6327 13.1803L17.8779 20.0902L12 15.8197L6.12215 20.0902L8.36729 13.1803L2.48944 8.90983H9.75486L12 2Z' fill='%23FFD805' /></svg>";
+$backgroundImageEmpty: "data:image/svg+xml;utf8,<svg width='24' height='24' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path d='M12 2L14.2451 8.90983H21.5106L15.6327 13.1803L17.8779 20.0902L12 15.8197L6.12215 20.0902L8.36729 13.1803L2.48944 8.90983H9.75486L12 2Z' fill='%23A3A3A3' /></svg>";
+
+.rating {
+  $star-size: var(--star-size);
+  $stars-gap: var(--stars-gap);
+  $section: calc($star-size + $stars-gap);
+  $bg-shift: calc($stars-gap / -2);
+  display: block;
+  position: relative;
+  width: calc(#{$section} * var(--max) - #{$stars-gap});
+  height: $star-size;
+  background-size: $section $star-size;
+  background-repeat: repeat-x;
+  background-position: $bg-shift 0;
+  border-radius: 0;
+  background-image: url($backgroundImageEmpty);
+
+  /* The bar in Firefox */
+  &:-moz-meter-optimum::-moz-meter-bar {
+    background-size: $section $star-size;
+    background-repeat: repeat-x;
+    background-position: $bg-shift 0;
+    background-image: url($backgroundImageColored);
+
+    opacity: 0.8;
+
+    &:hover {
```

Comment:
Here i added this hover effect only as example here: https://codesandbox.io/p/sandbox/quirky-sanderson-5xrrqj?file=/src/main.tsx:10,1
Please, check if in our design we need this.